Doc/build cache nano pico warning (#954)
CleverCloud/documentation: content/doc/administrate/apps-management.md

{{< callout type="info" >}}

>   If you set [`CC_DISABLE_BUILD_CACHE_UPLOAD`](/doc/develop/env-variables/#settings-you-can-define-using-environment-variables) environment variable to `true`, the cache archive won't be created nor uploaded.
> 
>   Nano and Pico plans always use a dedicated build, which requires the build cache. Setting this environment variable on these plans causes deployment to fail.

Reviewer: ```suggestion
If you set [`CC_DISABLE_BUILD_CACHE_UPLOAD`](/doc/develop/env-variables/#settings-you-can-define-using-environment-variables) environment variable to `true`, the cache archive won't be created nor uploaded. Don't use it with `nano` and `pico` plans which always use a build instance to create the build cache.
```
